redistimeseries/pom.xml

```diff
@@ -4,7 +4,7 @@
     <parent>
         <groupId>io.github.dengliming.redismodule</groupId>
         <artifactId>redis-modules-java</artifactId>
-        <version>1.0.1-SNAPSHOT</version>
+        <version>1.0.2-SNAPSHOT</version>
```

Comment:
Can you revert it?

Reply:
Reverted all pom.xml files.
Thanks. And congratulations, awesome work!